bundles/org.openhab.binding.bmwconnecteddrive/README.md

```diff
@@ -0,0 +1,686 @@
+
+Please note **this isn't a real-time binding**. 
+If a door is opened the state isn't transmitted and changed immediately. 
+This isn't a flaw in the binding itself because the state in BMW's own ConnectedDrive App is also updated with some delay. 
+
+## Supported Things
+
+### Bridge
+
+The bridge establishes the connection between BMW's ConnectedDrive Portal and openHAB.
+
+| Name                       | Bridge Type ID | Description                                                |
+|----------------------------|----------------|------------------------------------------------------------|
+| BMW ConnectedDrive Account | account        | Access to BMW ConnectedDrive Portal for a specific user    |
+
+
+### Things
+
+Four different vehicle types are provided. 
+They differ in the supported channel groups & channels. 
+Conventional Fuel Vehicles have no _Charging Profile_, Electric Vehicles don't provide a _Fuel Range_. 
+For hybrid vehicles in addition to _Fuel and Electric Range_ the _Hybrid Range_ is shown.
+ 
+| Name                                | Thing Type ID | Supported Channel Groups                               |
+|-------------------------------------|---------------|--------------------------------------------------------|
+| BMW Electric Vehicle                | bev           | status, range, location, service, check, charge, image |
+| BMW Electric Vehicle with REX       | bev_rex       | status, range, location, service, check, charge, image |
+| BMW Plug-In-Hybrid Electric Vehicle | phev          | status, range, location, service, check, charge, image |
+| BMW Conventional Vehicle            | conv          | status, range, location, service, check, image         |
+
+ 
+#### Properties
+
+<img align="right" src="./doc/properties.png" width="500" height="600"/>
+
+For each vehicle properties are available. 
+Basically 3 types of information are registered as properties
+
+* Informations regarding your dealer with address and phone number
+* Which services are available / not available
+* Vehicle properties like color, model type, drive train and construction year
+
+In the right picture can see in *Services Activated* e.g. the *DoorLock* and *DoorUnlock* services are mentioned. 
+This ensures channel group [Remote Services](#remote-services) is supporting door lock and unlock remote control.
+
+In  *Services Supported* the entry *LastDestination* is mentioned.
+So it's valid to connect channel group [Last Destinations](#destinations) in order to display and select the last navigation destinations.
+
+| Property Key       | Property Value      |  Supported Channel Groups    |
+|--------------------|---------------------|------------------------------|
+| Services Supported | Statistics          | last-trip, lifetime          |
+| Services Supported | LastDestinations    | destinations                 |
+| Services Activated | _list of services_  | remote                       |
```

Comment:
Property keys shouldn't contain white spaces and start with lower case.

Reply:
Fixed